meta-ostro/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Updates the EFI combo application
+BindsTo=update-triggers.target
+
+[Install]
+WantedBy=update-triggers.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
should this tool be in hidden namespace so it doesn't end in $PATH? @igor-stoppa 

also missing newline :-)